### PR TITLE
post: new blog about the themes livestream

### PR DESCRIPTION
**Pull request by jlengstorf**
Cross-posting the Gatsby themes livestream from last week to the Gatsby blog.

### CODE REVIEW

File: docs/blog/2019-02-11-gatsby-themes-livestream-and-example/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+1. How to build a Gatsby theme from scratch
+2. How to create [child themes](/blog/2019-01-29-themes-update-child-theming-and-component-shadowing/#child-theming)
+3. How to use [component shadowing](/blog/2019-01-29-themes-update-child-theming-and-component-shadowing/#component-shadowing)
+4. How to install and use a Gatsby theme
+
+The code we built is [available on GitHub](https://github.com/jlengstorf/livestream-gatsby-themes) — use it as a starting point to build your own Gatsby themes today!
+
+### Additional Links and Resources
+
+- Post: [Introducing Gatsby Themes](https://www.gatsbyjs.org/blog/2018-11-11-introducing-gatsby-themes/)
```
Comment (KyleAMathews) [inline docs/blog/2019-02-11-gatsby-themes-livestream-and-example/index.md]: Make these all relative links

**Comment by jlengstorf** [comment]
@gatsbyjs/core these test failures seem unrelated the blog post — is this something I need to fix?

File: docs/blog/2019-02-11-gatsby-themes-livestream-and-example/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+And _that’s it_.
+
+No code. No assembling plugins. Just writing content.
+
+## Learn how to build a Gatsby theme.
+
+Recently I had the pleasure of building a theme from scratch with Gatsby’s own [John Otander](https://twitter.com/4lpine) live on [my weekly livestream](https://twitch.tv/jlengstorf).
+
+<iframe id="ytplayer" type="text/html" width="720" height="405"
```
Comment (marcysutton) [inline docs/blog/2019-02-11-gatsby-themes-livestream-and-example/index.md]: Can you add a `title` attribute for this iframe to indicate where it goes (an accessibility best practice)? Something like "Theme livestream on YouTube" would work great.

Comment (jlengstorf) [reply docs/blog/2019-02-11-gatsby-themes-livestream-and-example/index.md]:
```suggestion
<iframe id="ytplayer" title="Theme livestream on YouTube" type="text/html" width="720" height="405"
```

Comment (jlengstorf) [reply docs/blog/2019-02-11-gatsby-themes-livestream-and-example/index.md]: @marcysutton good call — thanks!

**Pull request merged**